.github/workflows/merge-gate.yml

```diff
@@ -0,0 +1,46 @@
+name: Merge-Gate
+
+on:
+  workflow_call:
+    secrets:
+      ALTERNATIVE_GITHUB_TOKEN:
+        required: false
+
+jobs:
```

Comment:
Lowercase is usually more convenient. I use uppercase spelling in the other scenarios because, for example, for matrix builds, the main name used is the name of the step in the yaml file rather than the attribute of the `name` property.

Reply:
OK